File: src/main/java/nox/platform/PlatformInfoHolder.java
/**
 * Copyright (c) Profidata AG 2017
 */
package nox.platform;

import com.google.common.collect.Maps;

import java.util.Map;

/**
 * PlatformInfoHolder as a root project extension holds platform relevant information
 * for the overall build in a singleton manner.
 */
public class PlatformInfoHolder {

	public static final String name = "platformInfoHolder";

	public final Map<String, String> bundleMappings = Maps.newConcurrentMap();
}

File: src/main/kotlin/nox/core/PlatformInfoHolder.kt
/**
 * Copyright (c) Profidata AG 2017
 */
package nox.core


/**
 * PlatformInfoHolder as a root project extension holds platform relevant information
 * for the overall build in a singleton manner.
 */
open class PlatformInfoHolder {

	val bundleMappings = mutableMapOf<String, String>()

	companion object {

		val name = "platformInfoHolder"
	}
}
